internal/envoy/listener_test.go

```diff
@@ -149,10 +149,10 @@ func TestSocketAddress(t *testing.T) {
 
 func TestDownstreamTLSContext(t *testing.T) {
 	const (
-		cert = "foo"
-		key  = "secret"
+		secret  = "secret"
```

Comment:
please use

secretName = "default/tls-cert"
clusterName = "contour"

Reply:
that's a good call (test with the "namespace/name" variant)